src/components/POLine/POLineVersionView/DonorsVersionView/DonorsVersionView.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types';
+import {
+  useCallback,
+  useContext,
+  useMemo,
+} from 'react';
+
+import {
+  DonorsListContainer,
+  VersionViewContext,
+} from '@folio/stripes-acq-components';
+import { TextLink } from '@folio/stripes/components';
+
+import { getDonorUrl } from './utils';
+
+export const DonorsVersionView = ({ version }) => {
+  const donorOrganizationIds = version?.donorOrganizationIds;
+  const versionContext = useContext(VersionViewContext);
+
+  const changedDonorIds = useMemo(() => {
+    const changedDonors = versionContext?.changes?.filter((change) => {
+      return change.path.startsWith('donorOrganizationIds');
+    });
+    const donorIds = changedDonors.map(({ values }) => values.filter(Boolean));
+
+    return donorIds.flat();
+  }, [versionContext?.changes]);
+
+  const renderKeyValueComponent = useCallback(({ value, id, asLink = false }) => {
+    const content = asLink ? <TextLink to={getDonorUrl(id)}>{value}</TextLink> : value;
```

Comment:
Do you need to check if the url is returned from getDonorUrl method?

Reply:
I not checking. I am passing boolean to check if I need to return TextLink with value or not

Reply:
Which Boolean are you referring in your comment?